### PR TITLE
Documentation updates.

### PR DESCRIPTION
**Pull request by oliverchang**
Split out more workflow files to allow more flexibility with scheduling
tasks.

Add an overview diagram.

Delete advanced topics, generic documentation pages.

Various other rewording / fixes.

### CODE REVIEW

File: docs/running_clusterfuzzlite.md [hunk 1]
```diff
@@ -98,4 +95,9 @@ value is `false`. When set to `true`, ClusterFuzzLite will never report a
 failure even if it finds a crash in your project. This requires the user to
```
[the remark this answers is not in the document]
Comment (oliverchang) [reply docs/running_clusterfuzzlite.md:95]: I did this in https://github.com/google/oss-fuzz/pull/6600/files.

[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply docs/running_clusterfuzzlite.md:95]: Not sure to be honest.  I guess it was more of an item to think about than actually do. Sorry.

File: docs/running-clusterfuzzlite/github_actions.md [hunk 2]
```diff
@@ -30,11 +28,11 @@ TODO: multiple sanitizers
 Continuous builds are used whenever a crash is found during PR or batch fuzzing
 to determine if this crash was newly introduced.
 
-Add the following to `cflite_continuous.yml`:
+Add the following to `.github/workflows/cflite_build.yml`:
 
 {% raw %}
 ```yaml
-name: CIFuzz continuous
+name: ClusterFuzz Lite continuous builds
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:34]: ClusterFuzzLite should be one word.

File: docs/running-clusterfuzzlite/github_actions.md [hunk 3]
```diff
@@ -57,11 +55,11 @@ whenever a new push is done to main/default branch.
 ## PR fuzzing
 
 To add a fuzzing workflow that runs on all pull requests to your project, add
-the following to `cflite_pr.yml`:
+the following to `.github/workflows/cflite_pr.yml`:
 
 {% raw %}
 ```yaml
-name: CIFuzz PR fuzzing
+name: ClusterFuzz Lite PR fuzzing
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:61]: Same comment as above.

File: docs/running-clusterfuzzlite/github_actions.md [hunk 4]
```diff
@@ -73,29 +71,38 @@ jobs:
...
 
 To enable batch fuzzing, add the following to
-`cflite_continuous.yml`:
+`.github/workflows/cflite_batch.yml`:
+
+This can be run on either pushes to your default branch, or on a cron schedule (or both).
 
 {% raw %}
 ```yaml
+name: ClusterFuzz Lite batch fuzzing
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:96]: Same comment as above.

File: docs/running-clusterfuzzlite/github_actions.md (hunk 2 above)
Comment (oliverchang) [reply docs/running-clusterfuzzlite/github_actions.md:34]: done.

File: docs/running-clusterfuzzlite/github_actions.md (hunk 3 above)
Comment (oliverchang) [reply docs/running-clusterfuzzlite/github_actions.md]: done.

File: docs/running-clusterfuzzlite/github_actions.md (hunk 4 above)
Comment (oliverchang) [reply docs/running-clusterfuzzlite/github_actions.md]: done.

**Pull request merged**